app/containers/System/containers/Firmware/index.js

```diff
@@ -1,7 +1,178 @@
 import React from 'react';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+import { Alert, notification } from 'antd';
+import { GET_ALL_FIRMWARE, GET_TRACK_ASSIGNMENTS } from 'graphql/queries';
+import {
+  DELETE_TRACK_ASSIGNMENT,
+  DELETE_FIRMWARE,
+  CREATE_FIRMWARE,
+  UPDATE_FIRMWARE,
+} from 'graphql/mutations';
+import { Firmware as FirmwarePage, Loading } from '@tip-wlan/wlan-cloud-ui-library';
 
 const Firmware = () => {
-  return <h1>Firmware Page</h1>;
+  const { data, error, loading, refetch } = useQuery(GET_ALL_FIRMWARE);
+  const {
+    data: trackAssignmentData,
+    error: trackAssignmentError,
+    loading: trackAssignmentLoading,
+    refetch: refetchAssignmentData,
+  } = useQuery(GET_TRACK_ASSIGNMENTS);
+
+  const [deleteTrackAssignment] = useMutation(DELETE_TRACK_ASSIGNMENT);
+  const [deleteFirmware] = useMutation(DELETE_FIRMWARE);
+  const [createFirmware] = useMutation(CREATE_FIRMWARE);
+  const [updateFirmware] = useMutation(UPDATE_FIRMWARE);
```

Comment:
move Delete to the bottom, things should be listed in the order of CRUD for ease of readability